src/components/graph-view.js

```diff
@@ -152,15 +140,14 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
   renderNodesTimeout: any;
   renderEdgesTimeout: any;
   zoom: any;
-  viewWrapper: React.RefObject<HTMLDivElement>;
-  graphSvg: React.RefObject<SVGElement>;
+  viewWrapper: any;
+  graphSvg: any;
   entities: any;
-  graphControlsWrapper: React.RefObject<HTMLDivElement>;
-  highlightAreaRef: React.RefObject<SVGElement>;
+  graphControlsWrapper: any;
+  highlightAreaRef: any;
```

Comment:
The `React.RefObject` wasn't something that Flowtype supported, and flowtype's not great with SVG types, so instead of wasting time trying to figure out the perfect type, I defaulted these to any. If someone wants to help type this file better, then I would appreciate a later update.